source/WesternStatesWater.WaDE.Common/Ogc/OgcFeature.cs

```diff
@@ -0,0 +1,19 @@
+using System.Text.Json.Serialization;
+using NetTopologySuite.Features;
+using NetTopologySuite.Geometries;
+
+namespace WesternStatesWater.WaDE.Common.Ogc;
+
+/// <summary>
+/// Represents a GeoJSON Feature object around the OGC API Specification.
+/// </summary>
+/// <remarks>Due to OGC API specification of having links on the Feature. The current NetTopologySuite GeoJson converters
+/// do not support foreign members.</remarks>
```

Comment:
I asked GPT to make this remark a bit more dev friendly. Thoughts?

```suggestion
/// The OGC API specification requires that features include links (e.g., for navigation or related resources).
/// However, the current GeoJSON converters provided by the NetTopologySuite library do not support
/// adding extra custom properties (referred to as "foreign members") to the JSON output.
```